src/items/associated-items.md

```diff
@@ -279,6 +335,68 @@ impl<T> Container for Vec<T> {
 }
 ```
 
+### Required where clauses on generic associated types
+
+Generic associated type declarations on traits currently may require a list of
+where clauses, dependent on functions in the trait and how the GAT is used.
```

Comment:
I feel we should indicate somewhere that these rules are "in flux". Perhaps we could link to [the page on the generic-associated-types initiative repository](https://rust-lang.github.io/generic-associated-types-initiative/explainer/required_bounds.html)? Alternatively, isn't there an issue that we redirect people to?

Reply:
good idea